cmd/kops/create_cluster.go

```diff
@@ -354,6 +357,12 @@ func (c *CreateClusterCmd) Run(args []string) error {
 		fmt.Println("Previewing changes that will be made:\n")
 	}
 
+	if c.NoPublicIP {
+		fullCluster.Spec.AssociatePublicIP = fi.Bool(false);
```

Comment:
I think it would be better if the flag was named similarly to the spec option; the flags are meant as shortcuts and I'd like to keep their names as similar as possible to help people discover this.  So you would pass `--associate-public-ip=false` (true would be the default).

WDYT?